src/components/auth/Register.tsx

```diff
@@ -9,21 +9,21 @@ import { registerSchema } from 'schemas/auth';
 
 import FormButton from './FormButton';
 import FormField from './FormField';
-import { AuthFormProps, RegisterValue } from './interfaces';
+import { AuthFormProps, RegisterValues } from './interfaces';
 
-const initialValues: RegisterValue = {
+const initialValues: RegisterValues = {
```

Comment:
Basically the same logic for these values on register and login, but their names aren't consistant ("values" and "payload" in login and register accordingly). Replace login's with the second one